packages/react-core/src/components/Nav/NavItem.tsx

```diff
@@ -193,8 +193,8 @@ export const NavItem: React.FunctionComponent<NavItemProps> = ({
         {...(hasFlyout && { ...ariaFlyoutProps })}
         {...props}
       >
-        {hasNavLinkWrapper ? <span className={css(`${styles.nav}__link-text`)}>{children}</span> : children}
-        {flyout && flyoutButton}
```

Comment:
This will need to be added back in. Right now the Flyout example doesn't render the icon to indicate the item has any sort of interaction (item "Flyout link 3" in the following screenshot):

![image](https://github.com/patternfly/patternfly-react/assets/70952936/00fff6ee-c508-4052-b790-75e48df65bb8)

Reply:
oops that was a paste mistake!  good catch!